content/blog/2020/05/2020-05-06-docker-agent-image-renaming.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Renaming of official Docker agent images"
+tags:
+- announcement
+- docker
+- platform-sig
+author: oleg_nenashev
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+We would like to announce renaming of the official Docker images for Jenkins agents.
```

Comment:
I would change `We would like to announce renaming` to `We would like to announce the renaming`

Reply:
```suggestion
We would like to announce the renaming of the official Docker images for Jenkins agents.
```